src/client/java/daos-java/src/main/java/io/daos/BufferAllocator.java

```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
```

Comment:
2018-2021

Reply:
Good catch - the current pre-commit doesn't handle .java, will try to update it.